vmp/provider.go

```diff
@@ -57,7 +57,7 @@ func Provider() *schema.Provider {
 			"vmp_cname":               resourceCname(),
 			"vmp_customer":            resourceCustomer(),
 			"vmp_customer_user":       resourceCustomerUser(),
-			"vmp_httplarge_re_policy": resourceRulesEngineV4Policy(),
```

Comment:
FYI: we gonna have three policy resources based on each platform.
So, it's okay for now the naming but eventually we need to include three different RE policies.

Reply:
@changgyu-oh  Sounds good, I wasn't sure if that was just a testing name but that is a good idea.